tools/cloud-build/python-integration-tests/test.py

```diff
@@ -0,0 +1,428 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import shutil
+import os
+import re
+import signal
+import socket
+import subprocess
+import sys
+import time
+import paramiko
+from collections import defaultdict
+
+def run_command(cmd: str, err_msg: str = None) -> subprocess.CompletedProcess:
+        res = subprocess.run(cmd, shell=True, universal_newlines=True, check=False,
+                         stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if res.returncode != 0:
+            raise subprocess.SubprocessError(f"{err_msg}:\n{res.stderr}")
+       
+        return res
+
+def parse_blueprint(file_path: str):
+    try:
```

Comment:
blueprint is valid yaml, please use `parseyaml` instead of regexing.

Reply:
Updated to use parseyaml